### PR TITLE
Add DODTLV2015

### PR DESCRIPTION
**Pull request by m1keil**

### CODE REVIEW

File: site/content/events/2015-telaviv/_cfp_date_end.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+October 15, 2014
```
Comment (uric) [inline site/content/events/2015-telaviv/_cfp_date_end.txt]: The date says 2014

File: site/content/events/2015-telaviv/_email_organizers.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+organizers-israel-2015@devopsdays.org
```
Comment (uric) [inline site/content/events/2015-telaviv/_email_organizers.txt]: same here

File: site/content/events/2015-telaviv/location/index.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+extension: html
+filter:
+- erb
+- markdown
+title: location
+layout: event_tlv
+dirty: true
+---
+### HP Software
```
Comment (uric) [inline site/content/events/2015-telaviv/location/index.html]: remove HP software

File: site/content/events/2015-telaviv/program/index.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+<div class="span-2">15:30-16:00</div>
+<div class="span-4 append-bottom box last"><strong>Open Space Slot 5</strong></div>
+<div class="span-2">16:00-16:30</div>
+<div class="span-4 append-bottom box last"><strong>Open Space Slot 6</strong></div>
+<div class="span-2">16:30</div>
+<div class="span-4 append-bottom last">Closing</div>
+
+
+</div>
+</div>
```
Comment (uric) [inline site/content/events/2015-telaviv/program/index.html]: All talks need to be removed. This is last year's TOC

File: site/content/events/2015-telaviv/workshops/index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  <div class="span-7 last">
+    <h4>Day 2 </h4>
+    <div class="span-2">15:30-17:00</div>
+    <div class="span-4 append-bottom box last"><strong><a href="/events/2014-telaviv/proposals/Docker%20Workshop/">Next Generation Docker Based Development Environment - Adam Spektor</a></strong></div>
+      
+    <div class="span-2">15:30-17:00</div> 
+    <div class="span-4 append-bottom box last"><strong><a href="/events/2014-telaviv/proposals/InfluxDB%20Workshop/">InfluxDB, the open-source distributed time series database - Michael Sverdlik</a></strong></div>
+      
+  </div>
+</div>
```
Comment (uric) [inline site/content/events/2015-telaviv/workshops/index.html]: this also last year's agenda. You need to make everything Tbd

File: site/content/events/2015-telaviv/_cfp_date_end.txt (hunk 1 above)
Comment (m1keil) [inline site/content/events/2015-telaviv/_cfp_date_end.txt]: This file isn't used in the scripts

File: site/content/events/2015-telaviv/_email_organizers.txt (hunk 2 above)
Comment (m1keil) [inline site/content/events/2015-telaviv/_email_organizers.txt]: ? -2015

**Comment by m1keil** [comment]
I think better now?

File: site/content/events/2015-telaviv/_sponsors.txt [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+<% @csponsors.each do |sponsor| %>
+<a href='<%= sponsor[:link] %>'><img border=0 alt='<%= sponsor[:name] %>' title='<%= sponsor[:name] %>' width=100px height=100px src='<%= File.join('/',@eventhome,"logos/#{sponsor[:image]}")%>'></a>
+<% end %>
+
+<br/>
+<br/>
+
+<% @platinum_sponsors = [
+#	{ :image => "aolon.png", :name => "AOL On", :link => "http://on.aol.com/"},
```
Comment (uric) [inline site/content/events/2015-telaviv/_sponsors.txt]: Is AOL sponsoring this year or is it a leftover?

File: site/content/events/2015-telaviv/_sponsors.txt [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+
+<% @gold_sponsors.each do |sponsor| %>
+<a href='<%= sponsor[:link] %>'><img border=0 alt='<%= sponsor[:name] %>' title='<%= sponsor[:name] %>' width=80px height=80px src='<%= File.join('/',@eventhome,"logos/#{sponsor[:image]}")%>'></a>
+<% end %>
+
+<br/>
+<br/>
+
+<% @silver_sponsors = [
+#	{ :image => "github.png", :name => "GitHub", :link => "http://github.com"},
```
Comment (uric) [inline site/content/events/2015-telaviv/_sponsors.txt]: What about github?

File: site/content/events/2015-telaviv/_sponsors.txt [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+<% @silver_sponsors.each do |sponsor| %>
+<a href='<%= sponsor[:link] %>'><img border=0 alt='<%= sponsor[:name] %>' title='<%= sponsor[:name] %>' width=70px height=70px src='<%= File.join('/',@eventhome,"logos/#{sponsor[:image]}")%>'></a>
+<% end %>
+
+<br/>
+<br/>
+
+<% @community_sponsors = [
+#	{ :image => "igt.png", :name => "IGT", :link => "http://meetup.com/IGTCloud"},
```
Comment (uric) [inline site/content/events/2015-telaviv/_sponsors.txt]: remove

File: site/content/events/2015-telaviv/_sponsors.txt (hunk 6 above)
Comment (m1keil) [inline site/content/events/2015-telaviv/_sponsors.txt]: it's a comment, just a place holder to show how to fill this field.

File: site/content/events/2015-telaviv/_sponsors.txt (hunk 8 above)
Comment (m1keil) [inline site/content/events/2015-telaviv/_sponsors.txt]: see comment above (just a place holder for now... not visible on website.)

**Pull request merged**